src/components/AssetBreadcrumb/stories/helper.tsx

```diff
@@ -43,4 +43,15 @@ export const customElementRendering = (
 export const onBreadcrumbClick = (asset: Asset, depth: number) =>
   console.log(asset, depth);
 
+export const retrieveAsset = async (
+  assetId: CogniteInternalId
+): Promise<Asset> => {
+  return new Promise(resolve => {
```

Comment:
we agreed to use
```typescript
await sleep(100)
```
instead of wrapping `setTimeout()` into `Promise`.  Please, check similar mocks as a reference.